index.js

```diff
@@ -220,31 +224,41 @@ var appServer = function(config) {
                     if (privateKey != undefined && certificate != undefined) {
                         var credentials = { key: privateKey, cert: certificate };
 
-                        try { //The line below can fail it the certs were generated incorrectly. But we can continue startup without HTTPS
-                            self.httpsInstance = https.createServer(credentials, self.express).listen(config.httpsPort); //create the HTTPS server
-                            self.log("Listening on HTTPS port " + config.httpsPort);
+                        try { // These two lines below can fail it the certs were generated incorrectly. But we can continue startup without HTTPS
+                            var httpsServer = https.createServer(credentials, self.express); // create the HTTPS server
+
+                            //TODO: add separate option to specify specific host address for HTTPS server to bind to ???
```

Comment:
Tiny annoying nitpick. Can we add a space after `//`, so `// TODO:`, hurts me every time :)